package.json

```diff
@@ -34,5 +38,10 @@
   "bugs": {
     "url": "https://github.com/socraticorg/mathsteps/issues"
   },
-  "homepage": "https://github.com/socraticorg/mathsteps#readme"
+  "homepage": "https://github.com/socraticorg/mathsteps#readme",
+  "scripts": {
+    "requirements-check": "node scripts/check-version.js",
+    "preinstall": "npm run requirements-check",
+    "test": "mocha --recursive"
```

Comment:
you don't actually need to add this line :)

just add the ` "requirements-check"` and `"preinstall"` to the scripts starting on line 18